packages/cubejs-databricks-jdbc-driver/src/installer.ts

```diff
@@ -1,38 +1,32 @@
 import path from 'path';
 import { downloadAndExtractFile, getEnv } from '@cubejs-backend/shared';
 
-function acceptedByEnv() {
-  const acceptStatus = getEnv('databrickAcceptPolicy');
-  if (acceptStatus) {
-    console.log('You accepted Terms & Conditions for JDBC driver from DataBricks by CUBEJS_DB_DATABRICKS_ACCEPT_POLICY');
-  }
-
-  if (acceptStatus === false) {
-    console.log('You declined Terms & Conditions for JDBC driver from DataBricks by CUBEJS_DB_DATABRICKS_ACCEPT_POLICY');
-    console.log('Installation will be skipped');
-  }
-
-  return acceptStatus;
-}
-
+export const OSS_DRIVER_VERSION = '1.0.2';
+
+/**
+ * In the beginning of 2025 Databricks released their open-source version of JDBC driver and encourage
+ * all users to migrate to it as company plans to focus on improving and evolving it over legacy simba driver.
+ * More info about OSS Driver could be found at https://docs.databricks.com/aws/en/integrations/jdbc/oss
+ * As of March 2025 To use the Databricks JDBC Driver (OSS), the following requirements must be met:
+ * Java Runtime Environment (JRE) 11.0 or above. CI testing is supported on JRE 11, 17, and 21.
+ */
 export async function downloadJDBCDriver(): Promise<string | null> {
-  const driverAccepted = acceptedByEnv();
-
-  if (driverAccepted) {
-    console.log('Downloading DatabricksJDBC42-2.6.29.1051');
+  // TODO: Just to throw a console warning that this ENV is obsolete and could be safely removed
+  getEnv('databrickAcceptPolicy');
 
-    await downloadAndExtractFile(
-      'https://databricks-bi-artifacts.s3.us-east-2.amazonaws.com/simbaspark-drivers/jdbc/2.6.29/DatabricksJDBC42-2.6.29.1051.zip',
-      {
-        showProgress: true,
-        cwd: path.resolve(path.join(__dirname, '..', 'download')),
-      }
-    );
+  console.log(`Downloading databricks-jdbc-${OSS_DRIVER_VERSION}-oss.jar`);
 
-    console.log('Release notes: https://databricks-bi-artifacts.s3.us-east-2.amazonaws.com/simbaspark-drivers/jdbc/2.6.29/docs/release-notes.txt');
+  await downloadAndExtractFile(
+    `https://repo1.maven.org/maven2/com/databricks/databricks-jdbc/${OSS_DRIVER_VERSION}-oss/databricks-jdbc-${OSS_DRIVER_VERSION}-oss.jar`,
+    {
+      showProgress: true,
+      cwd: path.resolve(path.join(__dirname, '..', 'download')),
+      noExtract: true,
+      dstFileName: `databricks-jdbc-${OSS_DRIVER_VERSION}-oss.jar`,
+    }
+  );
 
-    return path.resolve(path.join(__dirname, '..', 'download', 'DatabricksJDBC42.jar'));
-  }
+  console.log(`Release notes: https://mvnrepository.com/artifact/com.databricks/databricks-jdbc/${OSS_DRIVER_VERSION}-oss`);
 
-  return null;
+  return path.resolve(path.join(__dirname, '..', 'download', `databricks-jdbc-${OSS_DRIVER_VERSION}-oss.jar`));
```

Comment:
We should update name here:

https://github.com/cube-js/cube/blob/f49ebcd055f76a9a7a33e8b93bf65f19e7de4222/packages/cubejs-databricks-jdbc-driver/src/post-install.ts#L12

<img width="800" alt="image" src="https://github.com/user-attachments/assets/0677034c-2356-4be0-b6c7-f99e17ba5ebd" />

Reply:
Oh, thnx! I missed that place because the filename is different. So afaiu, this is not needed anymore too.